source/main.lua

```diff
@@ -104,47 +106,52 @@ function love.load()
 	
 	love.window.setTitle("Mars Lander " .. gstrGameVersion)
 
-	fun.LoadGameSettings()
-	love.window.setFullscreen(garrGameSettings.FullScreen) -- Restore full screen setting
-	
-	fun.AddScreen("MainMenu")
-	fun.ResetGame()
-	
-	-- capture the 'normal' mass of the lander into a global variable
-	gintDefaultMass = Lander.getMass(garrLanders[1])
-
 	-- stills/images
 	--! should make these numbers enums one day
-	garrImages[1] = love.graphics.newImage("/Assets/tower.png")
-	garrImages[2] = love.graphics.newImage("/Assets/gastank1.png")
-	garrImages[3] = love.graphics.newImage("/Assets/Background-4.png")
-	garrImages[4] = love.graphics.newImage("/Assets/engine.png")
-	garrImages[5] = love.graphics.newImage("/Assets/ship.png")
-	garrImages[6] = love.graphics.newImage("/Assets/gastank1off.png")
-	garrImages[7] = love.graphics.newImage("/Assets/building1.png")
-	garrImages[8] = love.graphics.newImage("/Assets/building2.png")
-	garrImages[9] = love.graphics.newImage("/Assets/apollo-11-clipart-9.png")
+	local newImage	= love.graphics.newImage

```

Comment:
This is much cleaner. I like.

Reply:
might need to be changed later. It would make more sense to have these local version of newImage and newSource somewhere on top of the file to be able to use them everywhere in the file instead of just inside love.load. 
but for the moment we don't really load stuff anywhere else (which is good) and if we use the asset loader or something similar later on, we wouldn't need to either.